spec/support/vault_server.rb

```diff
@@ -28,7 +28,11 @@ def initialize
       end
 
       io = Tempfile.new("vault-server")
-      pid = Process.spawn({}, "vault server -dev", out: io.to_i, err: io.to_i)
+      pid = Process.spawn(
+        { "VAULT_DEV_ROOT_TOKEN_ID" => "root" },
+        "vault server -dev",
```

Comment:
might be easier to pass it as command-line flag:

```suggestion
        "vault server -dev -dev-root-token-id=root",
```